### PR TITLE
Custom tensormaps used for the whole-body MRI adiposity project

### PR DESCRIPTION
**Pull request by mklarqvist**

### CODE REVIEW

File: ml4h/normalizer.py [hunk 1]
```diff
@@ -28,8 +27,37 @@ def un_normalize(self, tensor: np.ndarray) -> np.ndarray:
...
+class NonZeroNormalize(Normalizer):
+    def normalize(self, tensor: np.ndarray) -> np.ndarray:
+        nonzero = tensor > 0
+        tensor[nonzero] = (tensor[nonzero] - tensor[nonzero].mean() + 1e-9) / (
+            tensor[nonzero].std() + 1e-9
+        )
+        return tensor
+
+
+class Top50Normalize(Normalizer):
```
Comment (ndiamant) [inline ml4h/normalizer.py:45]: maybe make more general by making `TopKNormalize`

File: ml4h/tensormap/ukb/mri.py [hunk 2]
```diff
@@ -2101,3 +2103,364 @@ def fallback_tensor_from_file(tm, hd5, dependents={}):
...
+        x2[i][:, midpoint:, :]  = data[i][:, midpoint:, :]  * weights2[i] + data[index2[i]][:, midpoint:, :]  * (1 - weights2[i])
+    return x2
+
+
+def mdrk_projection_single(field: str, instance: int = 2, augment: bool = False):
+    def _mdrk_projection_single(tm, hd5, dependents={}):
+        try:
+            compressed_data = hd5["instance"][str(instance)][field]
+        except Exception as e:
+            raise Exception(e)
```
Comment (ndiamant) [inline ml4h/tensormap/ukb/mri.py:2160]: Same thing as letting the exception happen?

**Review by ndiamant — COMMENTED** [review]
Very clean, good comments. 
What's the motivation behind this exception handling?
```
        try:
            ...
        except Exception as e:
            raise Exception(e)
```

**Comment by mklarqvist** [comment]
> Very clean, good comments.
> What's the motivation behind this exception handling?
> 
> ```
>         try:
>             ...
>         except Exception as e:
>             raise Exception(e)
> ```

Legacy reasons. Now removed.

File: ml4h/normalizer.py (hunk 1 above)
Comment (mklarqvist) [reply ml4h/normalizer.py]: Good idea! Implemented

**Review by ndiamant — APPROVED** [review]
Did you run pre-commit? If so, go ahead and merge

**Pull request merged**